bin.src/showDither.py

```diff
@@ -69,13 +74,13 @@ def main(butler, visits, fields, fieldRadius, showCCDs=False, aitoff=False, alph
         fieldRadius = np.radians(fieldRadius)
         dec = np.radians(dec)
         ra = np.array(ra)
-        ra  = np.radians(np.where(ra > 180, ra - 360, ra))
+        ra = np.radians(np.where(ra > 180, ra - 360, ra))
 
     plots, labels = [], []
     for v, r, d in zip(visits, ra, dec):
         field = fields.get(v)
         if verbose:
-            print "Drawing %s %s         \r" % (v, field),
+            print("Drawing %s %s         \r" % (v, field), end=' ')
```

Comment:
In that case, there are a few other places where stdout is flushed, so maybe you could quickly apply that there too?